src/action.js

```diff
@@ -93,9 +93,6 @@ ${changedExcludedPackages.join(', ')}. Skipping.`)
 function isAMajorReleaseBump(change) {
   const from = change.delete
   const to = change.insert
-  if (!from || !to) {
-    return false
-  }
```

Comment:
I wouldn't have expected functional changes in this PR. Why did we change the behavior here?

Reply:
I know doing functional changes on a test PR is not ideal, but in this case this path was never going to be reached.

The only place where this method is being called is to check for a major bump in the github-action-merge-dependabot package itself, but it will only fall on this path if the package has no old or new version (e.g. when being added or removed instead of updated). 
Will this case ever happen? Can dependabot add or remove dependencies?

Reply:
tbh I'm not sure it could ever be reached, I guess it was put there as a more defensive approach but I'm not really seeing how it could happen in a real world scenario.

I still can put it back and try to force this scenario through the tests tho, I'm just not sure it's going to be a problem

Reply:
why not? I mean the original code was parsing the title of the PR. if it couldn't it would have turned into either or both "from" and "to" to be falsy, and hence hit this condition

Reply:
I see what do you mean. I think before diff PR this would indeed be used.

One thing is if it branches on that if and it returns false it will actually approve the pull request, which seems a bit wrong to me and that's what prompted me to do the other change which is throwing when there isn't a match in the PR title. 

What do you think about that? I think the default should be not to merge a PR in case something is wrong seeing it can end up merging breaking changes by mistake into a project.

I'll revert the functional changes I did and we can work on them on another PR if needed.

Reply:
absolutely, if we can't figure out what the PR is trying to bump we shouldn't do anything, or maybe even throw and error in fact

Reply:
Ok then. I removed those changes and will add them back on a new PR.